src/assets/styles/components/_card.css

```diff
@@ -1,6 +1,6 @@
 .card {
```

Comment:
The accent colour on these cards should be at the bottom. So the `padding-block-start` should be removed for resource cards and replace with `padding-block-end` to match the design.